dist/hint.js

```diff
@@ -19,23 +19,23 @@ angular.hint.onAny(function (data, severity) {
 
 },{"./loader.js":2,"angular-hint":3}],2:[function(require,module,exports){
 /**
- * @license AngularJS v1.3.0-build.3042+sha.76e57a7
- * (c) 2010-2014 Google, Inc. http://angularjs.org
+ * @license AngularJS v1.6.4-build.5322+sha.d96e58f
+ * (c) 2010-2017 Google, Inc. http://angularjs.org
```

Comment:
I just realized the original comment was about `dist/hint.js`. My responses were about `loader.js`.
`dist/hint.js` includes `loader.js` (among othr things).

I had accidentally commited `dist/hint.js` and the `loader.js` changes on different commits (although the `dist/hinst.js` changes are directly affected by the `loader.js` changes). I have now amended squashed the two commits into one. Sorry for the confusion :grin: